# Parameter set for Long_Corridor sequence of SubT-MRS dataset

# topic: /velodyne_points (set in launch file or command line)
deskew: false
max_range: 100.0
min_range: 0.5
voxel_size: 0.25
map_cleanup_radius: 300
max_points_per_voxelized_scan: 1800
min_points_per_voxelized_scan: 1500
planarity_threshold: 0.17
max_points_per_voxel: 1
initial_threshold: 2.0
min_motion_th: 0.1
max_num_iterations: 100
convergence_criterion: 0.0001